src/useGoogleOneTapLogin.ts

```diff
@@ -21,7 +21,9 @@ function callback({
   onSuccess?: IUseGoogleOneTapLogin['onSuccess'];
 }) {
   if (data?.credential) {
-    fetch(`${oauthEndpointURL}${data.credential}`)
+    const id_token = data.credential;
+    console.log("got id_token: " + id_token);
+    fetch(`${oauthEndpointURL}${id_token}`)
```

Comment:
You can keep this code the same way as it was before and just add the credential to the `id_token` in the success response.